### PR TITLE
Fix some miispellis in cluster/engine.go

### PR DESCRIPTION
**Pull request by fate-grand-order**
Signed-off-by: fate-grand-order <chenjg@harmonycloud.cn>

### CODE REVIEW

File: cluster/engine.go [hunk 1]
```diff
@@ -192,13 +192,13 @@ func (e *Engine) Connect(config *tls.Config) error {
 	e.httpClient = httpClient
 	e.url = url
 
-	// Use HTTP Client created above to create a dockerclient client
+	// Use HTTP Client created above to create a docker client
```
Comment (dongluochen) [inline cluster/engine.go]: `dockerclient` is referring to `github.com/samalba/dockerclient`. It's not typo.

File: cluster/engine.go [hunk 2]
```diff
@@ -192,13 +192,13 @@ func (e *Engine) Connect(config *tls.Config) error {
...
 	c, err := dockerclient.NewDockerClient(url.String(), config)
 	if err != nil {
 		return err
 	}
 
-	// Use HTTP Client used by dockerclient to create docker/api client
+	// Use HTTP Client used by docker client to create docker/api client
```
Comment (dongluochen) [inline cluster/engine.go]: Same as above.

Comment (fate-grand-order) [inline cluster/engine.go]: I am so sorry, I will update it.

**Comment by dongluochen** [comment]
LGTM

**Pull request merged**